api/RTCIceCandidate.json

```diff
@@ -535,7 +542,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false
+              "version_added": "126"
```

Comment:
@Elchi3 The `toJson` metho (further down) changed signature as shown to reflect spec. I THINK that means no user impacted compat change though right?

```
-  [Default] object toJSON();
+  RTCIceCandidateInit toJSON();
```

https://bugzilla.mozilla.org/show_bug.cgi?id=1322186

Reply:
I think you are right and we don't need to capture this in compat data.